### ISSUE TITLE
Insert into main resource tables the validated and clean data 

### ISSUE THREAD

**Issue by dbrownukk** [dbrownukk/EFD_v2#65]

**Comment by dbrownukk** [comment]
Set to Commited

**Comment by dbrownukk** [comment]
Now that the WealthGroupInterview and Assets have a Status I think the Validation process will set the status to Valid and lock down WealthGroupInterview as Valid and prevent further parse or validation. Will not need the extra set of tables and a big insert.

**Comment by dbrownukk** [comment]
Aha. Validate button does not do anything at the moment. The first pass of 'Validation' is done during the parse of the spreadsheet, matching the entered resource sub type with an appropriate resource sub type. If a match is found then the Status is set to Valid for that record. 

A drop down list enables non matched records to be matched and then the Status can be set to Valid (manual at the moment).

That is the 'Validate' process.

There is no copy from temp to clean tables - the status field handles this.

**Comment by daiClegg** [comment]
i will test that by uploading & parsing a spreadsheet that has some valid and some invalid resources.

**Comment by daiClegg** [comment]
Tested ok.

I tested invalid (typo), invalid (missing synonym), invalid (missing resource)and valid records in the spreadsheet.
All loaded as expected. 
I was able to switch to the Resource Subtype Module to add synonyms and new resources, then switch back to the WG Interview Module to allocate, all seamlessly.
I weas able to selcect the correct resource in the yWH inteview detail view.

Tested for various examples on different tabs - though not every value in every tab.

But this looks good from a validate pov.
Will close this right now, pending some UX & Spreadsheet format issues that are documented elsewhere.
